syntax="proto3";

enum MsgIdTypeServer {
	MSGID_SERVER_NONE				= 0;
	MSGID_SERVER_ERROR				= 1;

	/* LoginServer <-> GateServer */
	MSGID_T2L_CONNECT_REQUEST		= 200000;
	MSGID_L2T_CONNECT_RESPONSE		= 200001;
	MSGID_L2T_ENTER_GAME_NOTIFY		= 200002;

	/* GateServer <-> GameServer */
	MSGID_G2T_CONNECT_REQUEST		= 200100;
	MSGID_G2T_CONNECT_RESPONSE		= 200101;
}

message MsgT2LConnectRequest {
	int32 gate_server_id = 1;
 	bytes gate_server_ip = 2;
	int32 gate_server_port = 3;
}

message MsgL2TConnectResponse {
}

message MsgL2TEnterGameNotify {
	bytes account = 1;
	bytes session_code = 2;
}

